doc/application/application.rst

```diff
@@ -768,8 +767,6 @@ preferred, since it correctly handles dependencies between options.
 #. Set kernel configuration values using the following
    key commands:
 
-   * Use the arrow keys to navigate within any menu or list.
```

Comment:
Is this no longer supported? I think this should be left here.

Reply:
Oops, this was a mistake, fixed, thanks!